client/ayon_core/hosts/maya/plugins/publish/extract_obj.py

```diff
@@ -54,12 +68,13 @@ def process(self, instance):
                                        pointsShaded=1,
                                        polygonObject=1):
                 with lib.shader(members,
-                                shadingEngine="initialShadingGroup"):
+                                shadingEngine="initialShadingGroup") if strip_shader else nullcontext():  # noqa: E501
                     with lib.maintained_selection():
                         cmds.select(members, noExpand=True)
                         cmds.file(path,
                                   exportSelected=True,
                                   type='OBJexport',
+                                  op=';'.join(f"{key}={val}" for key, val in self.obj_options.items()),  # noqa: E501
```

Comment:
Would it be nicer to move this `op` formatting earlier instead of inline for readability?